### ISSUE TITLE
Consider another EventEmitter

### ISSUE THREAD

**Issue by binarykitchen** [thomaswelton/requirejs-google-analytics#10]
Hi Thomas

No offense but I recommend to drop your event emitter https://github.com/thomaswelton/bower-event-emitter in favor of this one https://github.com/Wolfy87/EventEmitter

This because yours comes without tests and is not often updated. The other EventEmitter is great and widely used.

Currently I have too many different Event Emitter implementations in my bower directory, bloating up the file size.

See https://github.com/metafizzy/outlayer/issues/11 to get an idea.

Do you think you could switch the Event Emitter? That would be great!

**Comment by thomaswelton** [comment]
I wouldn't be adverse to this. I use my EventEmitter in a few projects, but I was copied from a Gist and I can't get hold of the author to get License info. Which was an issue when I needed to use this on a commercial project.
It looks like the repo you suggested is the most popular EventEmmiter and MIT license. So I'd do the conversion next time I need to use this.

**Comment by binarykitchen** [comment]
@thomaswelton Yes please do and switch to that popular EventEmitter whenever you have time, thanks!
